charts/consul/templates/api-gateway-gatewayclassconfig.yaml

```diff
@@ -28,6 +28,18 @@ spec:
     {{- else }}
       http: 8500
     {{- end }}
+  {{- if .Values.apiGateway.managedGatewayClass.deployment }}
```

Comment:
👍 on making this all conditional so that it's backwards compatible with older api gateways when not defined.